FabLabKasse/faucardPayment/FAUcardPaymentThread.py

```diff
@@ -78,7 +79,8 @@ def __init__(self, dialog, amount, thread=QtCore.QThread.currentThread()):
         self.card_number = 0
         self.old_balance = 0
         self.new_balance = 0
-        self.amount = int(amount * 100)
+        self.amount = amount
```

Comment:
Amount has carefully been replaced with amount_cents. This variable is used throughout the whole payment procedure and amount is only used as a Decimal backup. You are definetely right in the point that we want a Decimal and I just not to enforce the usage by float_to_decimal.
At the moment i expect the amount to already be decimal (the prior conversion in faucard.py wasa removed)